modules/unsupported/pom.xml

```diff
@@ -210,6 +216,7 @@
 				<module>css</module>
 				<module>geojsonstore</module>
 				<module>polylabel</module>
+                <module>mbstyle</module>

```

Comment:
ah, it is included in this list of "unsupported" modules - please remove.

This list is built since some of these modules are used by downstream projects; but they have not yet met our QA/doc/maintainer requirements.